apps/gateway-ui/src/App.tsx

```diff
@@ -50,18 +49,21 @@ export const App = React.memo(function Admin(): JSX.Element {
           setLoading(false);
         });
     } else {
-      gateway
-        .fetchInfo()
-        .then((gatewayInfo) => {
-          setGatewayInfo(gatewayInfo);
-        })
-        .catch(({ message, error }) => {
-          console.error(error);
-          setError(message);
-        })
-        .finally(() => {
-          setLoading(false);
-        });
```

Comment:
I haven't tested but I'm pretty sure without this, `loading` will be stuck as true indefinitely.

Reply:
testing now